store/store.go

```diff
@@ -1190,6 +1200,15 @@ type buyError struct {
 	ErrorMessage string `json:"error_message"`
 }
 
+type storeError struct {
```

Comment:
You might consider adding a `func Error() string` to this (or storeErrors) so that it can be used as a native go error (not sure if that is useful though, have not looked at how its used yet).